### ISSUE TITLE
TypeError: unsupported operand type(s) for /: 'PosixPath' and 'NoneType' make: *** [Makefile:18 : fonts] Erreur 1

### ISSUE THREAD

**Issue by Corne2Plum3** [Corne2Plum3/Giphurs#4]
Well... Seems to happen with `make fonts` at the line `cd sources && gftools builder config.yaml`

```Traceback (most recent call last):
  File "/(...)/Giphurs/venv/bin/gftools", line 10, in <module>
    sys.exit(main())
             ^^^^^^
  File "/(...)/Giphurs/venv/lib/python3.11/site-packages/gftools/scripts/__init__.py", line 91, in main
    mod.main(args[2:])
  File "/(...)/Giphurs/venv/lib/python3.11/site-packages/gftools/builder/__init__.py", line 705, in main
    builder.build()
  File "/(...)/Giphurs/venv/lib/python3.11/site-packages/gftools/builder/_ninja.py", line 49, in build
    self.build_static()
  File "/(...)/Giphurs/venv/lib/python3.11/site-packages/gftools/builder/_ninja.py", line 310, in build_static
    instances = self._instance_ufo_filenames(path, designspace)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/(...)/Giphurs/venv/lib/python3.11/site-packages/gftools/builder/_ninja.py", line 299, in _instance_ufo_filenames
    ufo = Path(path).parent / fn
          ~~~~~~~~~~~~~~~~~~^~~~
TypeError: unsupported operand type(s) for /: 'PosixPath' and 'NoneType'
make: *** [Makefile:18 : fonts] Erreur 1
```

Got this with b00140e0deeacaa25a1dddbe93479a013b13fe84 and python version 3.11.2

**Comment by Corne2Plum3** [comment]
Removing all intermediate weights (500-800) from config.yaml and Giphurs.designspace and still happening

**Comment by Corne2Plum3** [comment]
Seems the bug is coming from otfautohint, after the font binary being genetated for the first time. The error occurs because the program doesn't find any BlueValues.

Btw they should take the following values and can be implemented directly in fontforge:

* Weight 400: `[-20 0 1040 1060 1480 1500]`
* Weight 900: `[-30 0 1040 1070 1480 1510]`

After adding the BlueValues, I get the following error:
```
ERROR: Font has neither StemSnapV nor StdVW!
```

So I tried to add those values to Fontforge, in the same way I added the BlueValues:
* Weight 400: `StemSnapV = [90 120 135]` and `StdVW = [90]`
* Weight 900: `StemSnapV = [162 271 295]` and `StdVW = [162]`

But still the same error... The worst part is, on the fontinfo.plist those values are here, for every weight (even the intermediate ones)...

Then the same should be fixed for StemSnapH and StdHW but idk what to put here...

**Comment by Corne2Plum3** [comment]
Somehow fixed with 295892d98afe775f5701292b6356c39d9f2f3013 where I added the BlueValues etc. directly in the sfd sources...
